web/partials/editGroup.html

```diff
@@ -17,7 +17,7 @@
 
                     <h3 class="title">{{ user.email|uppercase }}</h3>
                     <div class="buttons">
-                        <span class="btn btn-red btn-small" data-ng-click="removeUser(user.id)" title="Remove user '{{ user.email }}' from group '{{ group.name }}'"><i class="fa fa-user-times" aria-hidden="true"></i> Remove User from Group</span>
+                        <a class="btn btn-red btn-small" data-ng-click="removeUser(user.id)" title="Remove user '{{ user.email }}' from group '{{ group.name }}'"><i class="fa fa-user-times" aria-hidden="true"></i> Remove User from Group</a>
```

Comment:
@DesignyourCode Not sure why this was a span, other buttons seem to be a's. As the test was expecting a's, I changed the actual partial here. If there's a valid reason this was a span, let me know and I'll change it back and have the test look for the span instead.